### ISSUE TITLE
Place anchor tag for main region inside <main></main> tags

### ISSUE THREAD

**Issue by Detzner** [ATLAS-Illinois/Illinois-Framework-Theme#820]
in templates > region > region—content.html.twig, line 24, 
<a name="main-content" aria-label="Start of content region"></a>
should be placed inside the <main> tag, otherwise Inspector flags it as content outside a region
